core/src/main/java/com/zextras/carbonio/files/dal/repositories/interfaces/NodeRepository.java

```diff
@@ -306,4 +306,12 @@ Optional<Node> getNodeByName(
     String folderId,
     String nodeOwner
   );
+
+  List<Node> findNodesByOwner(String ownerId);
+
+  Optional<Node> findFirstByOwner(String ownerId);
+
+  void invertHiddenFlagNodes(List<Node> nodesToFlag);
```

Comment:
Add javadoc to these methods. It will be inherited by every concrete classes that will implement this interface